commit bac7935c9b0820252e03765f7c9fc622df5d1e1a
Author: Valentin David <me@valentindavid.com>
Date:   Mon Jun 23 19:02:53 2025 +0200

    linux: Build module for DRM_MSM
    
    This is needed for the Oneplus 6.

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index e764e4261..45bcee89d 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -680,6 +680,7 @@ case "$arch" in
     ;;
     aarch64)
         module DRM_PANFROST
+        module DRM_MSM
     ;;
 esac
 
